Automattic/jetpack
# Search package: Add constant version number and some global functions

#### Changes proposed in this Pull Request:
The Search package is still refrencing some Jetpack constants which need to be replace. The PR added a file `search.php`  in the root of the package and set it to be always loaded. 

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Go to a site with Jetpack Search subscription
* Ensure everything works as usual
* Ensure PHP tests pass `( cd projects/packages/search && composer phpunit )`
